cmd/terminal-to-html/terminal-to-html.go

```diff
@@ -9,8 +9,8 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/buildkite/terminal-to-html/v3"
-	"github.com/urfave/cli"
+	"github.com/buildkite/terminal-to-html"
```

Comment:
This doesn’t work yet, because we haven’t published a working, tagged, go module versioning compatible v3 on GitHub yet. There’s probably a way to make it accept the local directory in the mean time, but this works.

After #75 lands, we need to do this:

```suggestion
	"github.com/buildkite/terminal-to-html/v3"
```

Reply:
#75 is merged now!

Reply:
Yep, turns out we need to tag a release in order to test this, or I need someone with go knowledge to help me test this locally